docs/details/shards/GFX/BuiltinFeature.md

```diff
@@ -0,0 +1,75 @@
+This shard generates features with preset behavior, the behavior depends on the [Id](../../../enums/BuiltinFeatureId)
+
+## Built-in Feature Ids
+
+### Transform
+
+This feature implements basic world/view/projection transform.
+
+Here are the entry points that is defines and can be added as dependencies by your own features.
+
+#### `initLocalPosition` (vertex)
+
+Sets the `localPosition` (float4) global from the vertex position
+
+Sets the `localPosition` (float4) global from the vertex position
+
+#### `initScreenPosition` (vertex)
+
+Sets the `screenPosition` global (float4) to the transformed vertex position (`=proj*view*world*<vertex>`)
```

Comment:
```suggestion
Sets the `screenPosition` global (float4) to the transformed vertex position (`=proj*view*world*<vertex>`).
```

We should keep the punctuation consistent for our docs - a period is required at the end of every sentence.